stem/IO/kratos_model_part_io.py

```diff
@@ -0,0 +1,219 @@
+from typing import Sequence, Optional, Dict
+
+import numpy as np
+
+from stem.model_part import ModelPart, BodyModelPart
+
+MAPPER_GMSH_TO_KRATOS: Dict[int, Dict[str, Dict[str,str]]] = {
```

Comment:
this is not correct. The mapper should be based on the element type, the dimension, the body modelpart or condition/load model part

Reply:
e.g. analysistype determines if element should be UPw or Pw; body model part determines if it is a beam or a rod element; element type determines the amount of nodes, and dimension determins the dimension

Reply:
@aronnoordam

I will close this pull request because superseded in #78.